addons/mail/static/src/js/followers.js

```diff
@@ -239,30 +239,42 @@ var Followers = AbstractField.extend({
             })
             .then(this._reload.bind(this));
     },
+    /**
+     * Remove partners or channels from the followers
+     * @param {Array} [ids.partner_ids] the partner ids
+     * @param {Array} [ids.channel_ids] the channel ids
+     */
+    _messageUnsubscribe: function (ids) {
+        var self = this;
```

Comment:
`self` is useless here ; you can replace all occurrences of `self` but `this` in this function?